CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 <!-- next version -->
 
+## 0.106.0
+
+### 🧰 Bug fixes 🧰
+
+- `operator`: When there were multiple OpenTelemetry Collector, the dashboard doesn't allow to select them individually. (#3189)
+- `target allocator`: Fix collector to target allocator connection in clusters with proxy. (#3187)
+  On clusters with global proxy the collector might fail to talk to target allocator
+  because the endpoint is set to `<ta-service-name>:port` and therefore it will go to proxy
+  and request might be forwarded to internet. Clusters with proxy configure `NO_PROXY` to `.svc.cluster.local` so 
+  the calls to this endpoint will not go through the proxy.
+  
+
+### Components
+
+* [OpenTelemetry Collector - v0.106.1](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.106.1)
```

Comment:
@iblancasa 

> service: Update all metrics to include otelcol_ prefix to ensure consistency across OTLP and Prometheus metrics (https://github.com/open-telemetry/opentelemetry-collector/pull/9759) 

https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.106.0 

Could you please check if it affects the OCP dashboard?

Reply:
All the metrics used for the dashboard included the `otelcol_` prefix before.